java/src/test/java/org/rocksdb/InfoLogLevelTest.java

```diff
@@ -113,7 +114,7 @@ public void valueOf() {
    * @throws IOException if file is not found.
    */
   private String getLogContentsWithoutHeader() throws IOException {
-    final String separator = System.getProperty("line.separator");
+    final String separator = Environment.isWindows() ? "\n" : System.getProperty("line.separator");
```

Comment:
I don't see why you need this change? `System.getProperty("line.separator")` should return `"\n"` on Windows platforms. Have you tested that?